docs/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@urturn/docs",
```

Comment:
this is the only change: rename the package to be under the `@urturn` npm org. Shouldn't matter because this module is private.